Source/Scene/SkyAtmosphere.js

```diff
@@ -125,6 +162,15 @@ define([
     /**
      * @private
      */
+    SkyAtmosphere.prototype.colorCorrect = function() {
```

Comment:
For better encapsulation, make this a local function and pass `skyAtmosphere` to it.

See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#private-functions